app/javascript/components/pages/__test__/SightNew.test.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import Enzyme, { shallow, render } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import { StaticRouter as Router } from "react-router-dom";
+import SightNew from "../SightNew";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe("When sightNew renders", () => {
+  const mockTrip = {
+    trip_name: "Getting some sun",
+    trip_location: "Jamaica",
+    trip_date_range: "12/20/2021 - 1/5/2022",
+    is_public: false,
+    user_id: 1,
+    sights: [
+      {
+        name: "Taco Bell",
+        address1: "Bell Street",
+        city: "Taco",
+        zip_code: "11111",
+        state: "Jamaica Land",
+        country: "Jamaica",
+        phone: "222-222-2222",
+        trip_id: 1,
+      },
+    ],
+  };
+
+  const sightNew = shallow(<SightNew trip={mockTrip} />);
+
+  it("displays a heading", () => {
+    const title = sightNew.find("h1");
+    expect(title.text()).toEqual("Add A New Sight");
+  });
+
+  it("displays 7 labels", () => {
+    const labels = sightNew.find("Label");
+    expect(labels.length).toEqual(7);
+    expect(labels.debug().includes("Name of Sight")).toEqual(true);
+    expect(labels.debug().includes("Street Address")).toEqual(true);
+    expect(labels.debug().includes("City")).toEqual(true);
+    expect(labels.debug().includes("Zipcode")).toEqual(true);
+    expect(labels.debug().includes("State")).toEqual(true);
+    expect(labels.debug().includes("Country")).toEqual(true);
+    expect(labels.debug().includes("Phone Number")).toEqual(true);
```

Comment:
this is probably working just find, but using `.debug()` for permanent tests is weird because it's a function not meant for making assertions. Also, checking on includes this way doesn't assert on the order of the properties which may be important for the UI

Reply:
[.toEqual(true) can also be .tobeTurthy()](https://jestjs.io/docs/expect#tobetruthy)